docs/src/components/CombinationNew.js

```diff
@@ -0,0 +1,66 @@
+// @flow strict
+import { Flex } from 'gestalt';
+import React, { type Node } from 'react';
+import MainSectionCard from './MainSectionCard.js';
+
+type Props = {
+  children: (Object, number) => Node, // flowlint-line unclear-type:off
```

Comment:
We can improve this type (`Object` is to be avoided, and will be removed in future Flow versions) but that can be handled repo-wide in a separate PR, not important here

Reply:
Although a small change that would improve readability here would be to name the arguments (and we might as well fix the props type enough to obviate the Flow suppression):
```js
children: (props: { [key: string]: any, ... }, index?: number) => Node,
```